scss/tech.scss

```diff
@@ -1,13 +1,12 @@
 .techs {
-  display: block;
+  display: flex;
+  flex-wrap: wrap;
   margin: 0 auto;
-  max-width: 1000px;
+  max-width: 1200px;
 
   .tech {
-    display: inline-block;
     padding: 10px;
-    vertical-align: top;
-    width: 24%;
+    width: 25%;
```

Comment:
Try not to use percentages, they will make the elemet to loose its shape. Use fixed values in px

Reply:
Done